drivers/storage/portworx/component/securitycontextconstraints.go

```diff
@@ -99,11 +113,16 @@ func (s *scc) Reconcile(cluster *opcorev1.StorageCluster) error {
 			}
 		} else if err == nil {
 			resourceVersion := existingScc.ResourceVersion
-			pxutil.CleanupObject(&scc)
```

Comment:
I have a feeling we don't need to do a obj cleanup, do we? If we have a proper customized deep equal function, and setting the latest resource version, can't we update the priority and AllowPrivilegeEscalation directly?

Reply:
existingScc has some system generated artifacts like timestamps, which are not in scc, but for comparison, we want to ignore these fields. Are you suggesting to skip comparing those fields like timestamps

Reply:
yes, we are using our customized deep equal function anyway, we can do the comparison without those fields, so we don't need to reset anything.

Reply:
Make sense to me.